api/apiFunctions.interface.ts

```diff
@@ -8,6 +8,15 @@ export interface IAccountData {
   password: string;
 }
 export interface IUser {
+  // for the appwrite auth collection
+  id: string;
+  email: string;
+  leagues: string[];
+  labels: string[];
+}
+
+export interface ICollectionuser {
+  // for the custom user collection
```

Comment:
I'd love to see these as JSDocs instead of regular comments so I can refer to them again if possible please :)

Reply:
@vmaineng I will probably defer this to an other time since we are not using JSDocs in the interface files to keep it consistent.